templates/alarms/template.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-template
+kind: Template
+metadata:
+  name: terraform-alarms
+  title: Create an alarm to notify Slack channel of failures 
+  description: |
+    Creates the terrafrom code and infrastructure to add alarms to your project. 
+
+  tags:
+    - terraform
+    - aws
+    - alarms
+    - add
+spec:
+  owner: group:cds-snc/internal-sre
+  type: service
+
+  # These parameters are used to generate the input form in the frontend, and are
+  # used to gather input data for the execution of the template.
+  parameters:
+    - title: Fill in the Alarm Details
+      required:
+        - cloudwatch_log_group
+        - slack_webhook_url 
+        - error_alarm_name
+        - error_alarm_description
+        - warning_alarm_name
+        - warning_alarm_description
+        - billing_tag_value 
+      properties:
+        cloudwatch_log_group:
+          title: Cloudwatch Log Group
+          type: string
+          description: The Cloudwatch log group that you want to monitor, such as /aws/lambda/log-system-api. Required.
+          ui:autofocus: true
+        error_alarm_name:
+          title: Error Alarm Name
+          type: string
+          description: The name of the error alarm. Required.
+          ui:autofocus: true
+        error_alarm_description:
+          title: Error Alarm Description
+          type: string
+          description: The description of the error alarm. Required.
+          ui:autofocus: true
```

Comment:
What would you think about instead of having the alarm name and description, we just captured the service name.  Then we could automatically generate all the alarm names and descriptions based on the service name and thresholds:
```yaml
# Error alarm name
${{ values.service_name | dump }}-Errors

# Error alarm description
${{ values.service_name | dump }} errors >= ${{ values. error_threshold | dump }} in 1 minute
```

Reply:
Good idea Pat! I will make the change.

Reply:
Done! Thanks for the suggestion - this is so much better than I originally had it implemented.

Reply:
Quotes added. Thanks Pat - I should have caught this.

Reply:
All good - it's tough to catch stuff like this without the TF plan running.